document/core/appendix/properties.rst

```diff
@@ -59,16 +59,12 @@ Results
    }
 
 
-:ref:`Results <syntax-result>` :math:`\val^\ast~(\THROWadm~\tagaddr)`
+:ref:`Results <syntax-result>` :math:`\XT[(\REFEXNADDR~a)~\THROWREF]`
```

Comment:
It feels kind of nasty to classify an instruction inside a context as a "result". Can you walk me through why you don't want to use something like `\THROWadm` any more?

Reply:
The introduction of the context is actually unrelated to replacing THROWadm, its absence was simply wrong before. We need it here since there is no reduction rule that can eliminate it on the outermost level (outside a function). We can't simply add a rule XT[exn] ⇝ exn, since that would introduce overlapping rules.

I see two ways to fix this in the future: (1) switching to a bubbling semantics for exceptions (avoiding overlap), or (2) introducing a separate global reduction relation like we have with threads. The former should go along with changes to other parts of the semantics to stay coherent, the latter I'd rather do after we have merged with threads. Hence I don't want to do either in this PR.